### ISSUE TITLE
Add Report Viewer

### ISSUE THREAD

**Issue by ChristopherPHolder** [ChristopherPHolder/app-speed#83]
## TL;DR

A user should be able to see the report using a viewer.

### Description

A feature lib that takes the report generated by lighthouse and displays the results. 

it should have a viewer for: 

- [ ] Metrics

![image](https://github.com/ChristopherPHolder/app-speed/assets/40126819/5010f5f3-7925-4012-a04c-663fd87f3c8d)

- [ ] Filmstrip

![image](https://github.com/ChristopherPHolder/app-speed/assets/40126819/585d28cd-ae43-4ae2-94a9-771104ac01cd)

- [ ] Diagnostics

![image](https://github.com/ChristopherPHolder/app-speed/assets/40126819/1c13f684-0550-4726-971d-a60b3a390bae)

- [ ] Passed Audits 

![image](https://github.com/ChristopherPHolder/app-speed/assets/40126819/83281138-1bed-476b-ba47-77b2ffd2c695)
